server/server/src/billing/app_store_client.rs

```diff
@@ -159,15 +159,15 @@ fn get_trans(
 ) -> Result<(LastTransactionItem, TransactionInfo), ServerError<UpgradeAccountAppStoreError>> {
     let last_trans = sub_group
         .last_transactions
-        .get(0)
+        .first()
         .ok_or(ClientError(UpgradeAccountAppStoreError::InvalidAuthDetails))?;
 
     let part = <&str>::clone(
         last_trans
             .signed_transaction_info
             .split('.')
             .collect_vec()
-            .get(1)
```

Comment:
should this have been `.first()`? I think that's `.get(0)`

Reply:
Yeah I think that was botched